src/Cfonb120Reader.php

```diff
@@ -37,6 +37,10 @@ public function __construct()
     /** @return Statement[] */
     public function parse(string $content): array
     {
+        if (!empty($content) && strlen($content) > 120 && strpos($content, "\n") === false) {
```

Comment:
I'm wondering if this lib should handle this feature. What about doing this string manipulation on your side? Do you think this should be to the library level?

Reply:
I have done this on my side for the moment, and i asked myself if it should be on library side or not.
At final i believe that this package should handle cfonb from all bank.
This should not be the responsibility to client to take this kind of weirdo.